### PR TITLE
Added answers to the question

### PR DESCRIPTION
**Pull request by sohmpatil**
@sanket8397 Please check and review the answers. Please decline if there is some change.

### CODE REVIEW

File: answers.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+*.jar
+*.war
+*.ear
+*.nar
+
+What does fast-forward in git mean? 
+Ans:Fast-forward merges literally move your main branch's tip forward to the end of your feature branch. This keeps all commits created in your feature branch sequential while integrating it neatly back into your main branch.
+
+Why should you have different branches? 
+Ans:For multiple people working in parallel, it is really necessary to track their work in different places and easier to merge them later to a single source. This helps a lot to working in parallel and it creates less confusion.
```
Comment (sanket8397) [inline answers.txt]: Check grammar and complete the answer again.

**Review by sanket8397 — CHANGES_REQUESTED** [review]

File: answers.txt (hunk 1 above)
Comment (sohmpatil) [reply answers.txt]: LGTM. Thanks for reviewing.

File: answers.txt [hunk 2]
```diff
@@ -11,7 +11,7 @@ What does fast-forward in git mean?
 Ans:Fast-forward merges literally move your main branch's tip forward to the end of your feature branch. This keeps all commits created in your feature branch sequential while integrating it neatly back into your main branch.
 
 Why should you have different branches? 
-Ans:For multiple people working in parallel, it is really necessary to track their work in different places and easier to merge them later to a single source. This helps a lot to working in parallel and it creates less confusion. 
+Ans:For multiple people working in parallel, it is really necessary to track their work at different places and easier to merge them later to a single source. This helps a lot to working in parallel and it creates less confusion.  
 
 In a project how many remote Git repositories do you usually have?
-Ans: There is usually only one major repository for the project, but many a times for some secondary purpose, people create new subprojects to work on them with separate teams which serve different purposes. Usually projects have only one repository in early stages. But as the time goes, many repositories are added to project to serve other purposes.
```
Comment (sanket8397) [inline answers.txt]: LGTM.

File: answers.txt [hunk 3]
```diff
@@ -11,7 +11,7 @@ What does fast-forward in git mean?
 Ans:Fast-forward merges literally move your main branch's tip forward to the end of your feature branch. This keeps all commits created in your feature branch sequential while integrating it neatly back into your main branch.
 
 Why should you have different branches? 
-Ans:For multiple people working in parallel, it is really necessary to track their work in different places and easier to merge them later to a single source. This helps a lot to working in parallel and it creates less confusion.
```
Comment (sanket8397) [inline answers.txt]: LGTM.

**Review by sanket8397 — APPROVED** [review]
LGTM.

**Pull request merged**